### PR TITLE
First working website code

### PR DESCRIPTION
**Pull request by jason5122**
I used an existing template that I use for my personal website, jason-han.com, to get started if that's okay.

# Changes
### Added
- 3 new pages (blog.html, projects.html, and resume.html)
      - Clickable JS to navigate
- Favicon
- Picture of me and a PDF
- CSS styles for each page as well as some global CSS

### Removed
- Message generator

# Screenshots

### About page
Screenshot 2020-06-04 at 3 45 51 PM

### Resume page
Screenshot 2020-06-04 at 3 48 43 PM

### Projects/Blog pages (under construction)
Screenshot 2020-06-04 at 3 46 09 PM

### CODE REVIEW

File: portfolio/src/main/webapp/blog.html [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  <link href="https://fonts.googleapis.com/css?family=Quicksand|Josefin+Sans:300|Raleway" rel="stylesheet">
+  <script defer src="https://use.fontawesome.com/releases/v5.0.13/js/all.js" integrity="sha384-xymdQtn1n3lH2wcu0qhcdaOpQwyoarkgLVxC/wZ5q7h9gHtxICrpcaSUfygqZGOe"
+          crossorigin="anonymous"></script>
+</head>
+
+<body>
+  <nav class="navbar">
+    <div class="navbar-wrapper">
+          <span class="navbar-toggle" id="js-navbar-toggle">
+              <i class="fas fa-bars"></i>
```
Comment (lyze) [inline portfolio/src/main/webapp/blog.html]: does this `i` tag do anything?

File: portfolio/src/main/webapp/blog.html [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+                  <a href="blog.html" class="nav-links">Blog</a>
+              </li>
+          </ul>
+        </div>
+      </nav>
+  <div id="wrapper">
+
+    <div class="main-content" style="width: 100%; text-align: center;">
+      <div class="box">
+        <p> This section is currently under development. </p>
```
Comment (lyze) [inline portfolio/src/main/webapp/blog.html]: nit: I would remove whitespace around tags.

File: portfolio/src/main/webapp/css/global.css [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+  font-size: 12px;
+
+  height: 65px;
+  width: 300px;
+  align-items: center;
+
+  margin: auto;
+  padding-left: 6%;
+  padding-right: 6%;
+  /* padding-top: 1%; */
```
Comment (lyze) [inline portfolio/src/main/webapp/css/global.css]: let's remove commented-out code

File: portfolio/src/main/webapp/css/global.css [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+  color: black;
+  transition:color linear 0.175s;
+}
+
+.footer a:hover {
+  color:#89BBE7;
+}
+
+@media screen and (max-width: 320px) {
+	.footer {
```
Comment (lyze) [inline portfolio/src/main/webapp/css/global.css]: indentation appears to be a bit off starting around here. Could you run a code formatter across the project? prettier.io is a good one to start with.

File: portfolio/src/main/webapp/projects.html [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+  <link href="https://fonts.googleapis.com/css?family=Quicksand|Josefin+Sans:300|Raleway" rel="stylesheet">
+  <script defer src="https://use.fontawesome.com/releases/v5.0.13/js/all.js" integrity="sha384-xymdQtn1n3lH2wcu0qhcdaOpQwyoarkgLVxC/wZ5q7h9gHtxICrpcaSUfygqZGOe"
+          crossorigin="anonymous"></script>
+</head>
+
+<body>
+  <nav class="navbar">
+    <div class="navbar-wrapper">
+          <span class="navbar-toggle" id="js-navbar-toggle">
+              <i class="fas fa-bars"></i>
```
Comment (lyze) [inline portfolio/src/main/webapp/projects.html]: same question here, is this `i` tag doing anything

File: portfolio/src/main/webapp/resume.html [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+  <link href="https://fonts.googleapis.com/css?family=Quicksand|Josefin+Sans:300|Raleway" rel="stylesheet">
+  <script defer src="https://use.fontawesome.com/releases/v5.0.13/js/all.js" integrity="sha384-xymdQtn1n3lH2wcu0qhcdaOpQwyoarkgLVxC/wZ5q7h9gHtxICrpcaSUfygqZGOe"
+          crossorigin="anonymous"></script>
+</head>
+
+<body>
+  <nav class="navbar">
+    <div class="navbar-wrapper">
+          <span class="navbar-toggle" id="js-navbar-toggle">
+              <i class="fas fa-bars"></i>
```
Comment (lyze) [inline portfolio/src/main/webapp/resume.html]: and here?

**Review by lyze — COMMENTED** [review]
Would you mind attaching screenshots or a gif? One tool to use is https://sites.google.com/corp/google.com/uxrtools/approved-tools/screencastify-2, and I think Ryan has been using it with some success.

File: portfolio/src/main/webapp/blog.html (hunk 1 above)
Comment (jason5122) [reply portfolio/src/main/webapp/blog.html]: Doesn't seem like it, it's probably some old code that I forgot to remove.

File: portfolio/src/main/webapp/projects.html (hunk 5 above)
Comment (jason5122) [reply portfolio/src/main/webapp/projects.html]: More old code!

File: portfolio/src/main/webapp/resume.html (hunk 6 above)
Comment (jason5122) [reply portfolio/src/main/webapp/resume.html]: Same as above `i` tags

File: portfolio/src/main/webapp/css/global.css (hunk 4 above)
Comment (jason5122) [reply portfolio/src/main/webapp/css/global.css]: Sure I'll give that a go

**Review by lyze — CHANGES_REQUESTED** [review]
Can you also run prettier on the css files?

Let's also squash these commits down before we merge.

I believe something like `git rebase -i master` should pull up an appropriate UI, or `git rebase -i HEAD~N` where `N` is the number of commits to look back.

http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html

File: portfolio/src/main/webapp/index.html [hunk 7]
```diff
@@ -1,18 +1,88 @@
...
+    <footer class="footer">
+      <img class="divider" src="images/divider.png" />
+      <div class="links">
+        <span><a href="mailto:jjhan@vassar.edu">Email</a></span>
+        <span>/</span>
+        <span
+          ><a href="https://www.linkedin.com/in/jasonhan-vassar/"
+            >Linkedin</a
+          ></span
+        >
```
Comment (lyze) [inline portfolio/src/main/webapp/index.html]: did prettier put these `>` on the next line?

**Review by lyze — CHANGES_REQUESTED** [review]
I would make sure to squash these commits before/when you merge into master.

File: portfolio/src/main/webapp/index.html (hunk 7 above)
Comment (jason5122) [reply portfolio/src/main/webapp/index.html]: it actually did, I just tried reproducing! I'll double check after formatting from now on

**Review by lyze — APPROVED** [review]

**Pull request merged**